### DISCUSSION TITLE
Only generate prompts and saving them to a file for future use

### DISCUSSION THREAD

**Discussion by avraham** [adieyal/sd-dynamic-prompts#258] [Q&A] [answered]
Hi,
Would it be possible to use this as a standalone tool independently of stable diffusion?

My idea is to have my collection of wildcards and define a Jinja template to only generate a batch of dynamic prompts to a file or printed as an output, so I can later use the generated (resolved) prompts manually on a different Stable diffusion setup.

This would be great to work and polish my dynamic prompts even if I don't have access to Stable diffusion.

Thank you.

**Comment by adieyal** [comment, marked as answer]
Yes. The extension is built on top of the dynamicprompts library (https://github.com/adieyal/dynamicprompts). Have a look at the sample code and let me know if that helps.

**Comment by avraham** [comment]
This looks exactly what I was looking for.
Thank you!
